wkcuber/wkcuber/downsample.py

```diff
@@ -0,0 +1,78 @@
+"""This module takes care of downsampling WEBKNOSSOS datasets."""
+
+from argparse import Namespace
+from multiprocessing import cpu_count
+from pathlib import Path
+from typing import Annotated, Optional, Union
+
+import typer
+from rich import print as rprint
+
+from webknossos import Dataset, SamplingModes
+from webknossos.utils import get_executor_for_args
+from wkcuber.utils import DistributionStrategy, SamplingMode
+
+app = typer.Typer(
+    invoke_without_command=True,
+)
+
+
+@app.callback()
+def main(
+    *,
+    target: Annotated[
+        Path,
+        typer.Argument(help="Path to your WEBKNOSSOS dataset.", show_default=False),
+    ],
+    sampling_mode: Annotated[
+        SamplingMode, typer.Option(help="The sampling mode to use.")
+    ] = SamplingMode.ANISOTROPIC,
+    layer_name: Annotated[
```

Comment:
Is this optional? What happens if I want to downsample a whole dataset with several layers? Do I have to call this command repeatedly for each layer? Or is the default to downsample everything?

Reply:
Yes it is optional